.github/workflows/desktop-build.yml

```diff
@@ -208,28 +208,28 @@ jobs:
 
           - target: 10.14_Mojave
             os: macos-10.15 # runs on Catalina
-            xcode: 10.3 # allows compatibility with macOS 10.14
+            xcode: 11.3 # allows compatibility with macOS 10.14
             type: Release
             do_tests: 0
             make_package: true
 
           - target: 10.15_Catalina
             os: macos-10.15
-            xcode: 12.1
+            xcode: 12.4
             type: Release
             do_tests: 0
             make_package: true
 
           - target: 11_Big_Sur
             os: macos-11
-            xcode: 12.5.1
+            xcode: 13.2
```

Comment:
This will no longer target 11, but 12.1

Xcode 13.0 is the max there.

Reply:
I found the last supported version at https://developer.apple.com/support/xcode/ which led me to changing these values

Reply:
GitHub only lists the supported SDK's for their runners without the minimum OS (which are actually lower it seems):
https://github.com/actions/virtual-environments/blob/main/images/macos/macos-10.15-Readme.md#installed-sdks
https://github.com/actions/virtual-environments/blob/main/images/macos/macos-11-Readme.md#installed-sdks

Looks like those upgrades are fine then. Double checked again.
Thanks for sharing the link!! 👍